src/app/login/login.component.html

```diff
@@ -0,0 +1,19 @@
+<div style="height: 60%; width: 60%; margin-left: 10%; margin-top: 11%; margin-bottom: 9%;">
+  <form (submit)="login()">
+    <div class="mb-3">
+      <label for="exampleInputEmail1" class="form-label">Email address</label>
+      <input name="name" type="email" class="form-control"  [(ngModel)]="user.email" id="exampleInputEmail1" aria-describedby="emailHelp">
+
+    </div>
+    <div class="mb-3">
+      <label for="exampleInputPassword1" class="form-label">Password</label>
+      <input name="passsword" type="password" [(ngModel)]="user.password" class="form-control" id="exampleInputPassword1">
+    </div>
+
+    <button type="submit" (click)="login()" class="btn">Login</button>
+    <p>New to Maveric Book? <a routerLink="/signup">Signup</a></p>
+    <p><a href="#">Forgot Password</a></p>
+  </form>
+
+  <h5 *ngIf="successMsg" style="color: green;">{{successMsg}}</h5>
```

Comment:
use p tag instead of h5 for avoiding accessibility issues

Reply:
done